spec/system/update_password_modal_system_spec.rb

```diff
@@ -73,7 +73,7 @@
 
   def fill_modal(password_attrs)
     within('div.modal-body') do
-      expect(page).to have_css('input', count: 3)
+      expect(page).to have_css('input')
```

Comment:
why did that change from 3 to 1 ?

Reply:
I changed that back to assert  3 input fields.